content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -217,6 +217,8 @@ that the CRI integration plugin is disabled by default.
 You need CRI support enabled to use containerd with Kubernetes. Make sure that `cri`
 is not included in the`disabled_plugins` list within `/etc/containerd/config.toml`;
 if you made changes to that file, also restart `containerd`.
+
+If you experience container crash loops after the initial cluster installation or after installing a CNI, the containerd configuration provided with the package might contain incompatible configuration parameters. Consider resetting the containerd configuration with <code>containerd config default > /etc/containerd/config.toml</code> as specified in [getting-started.md](https://github.com/containerd/containerd/blob/main/docs/getting-started.md#advanced-topics) and then set the configuration parameters specified above accordingly.
```

Comment:
@ramobis, please wrap this text to match the rest of the Markdown in the source file.

Reply:
@sftim, thank you for the proposal.
I applied the change accordingly.

Reply:
Somehow the checks are now failing.. But I can not view the logs. The deploy log just loads forever.

Reply:
@ramobis can you update the PR branch to re-trigger test run?

Reply:
Hi @bart0sh
The PR branch is now up to date.